tests/user/test_jwt_utils.py

```diff
@@ -16,6 +17,13 @@ def boto3_mock() -> Generator[Mock, None, None]:
         yield mock
 
 
+@pytest.fixture(autouse=True)
+def jwt_datetime_mock() -> Generator[Mock, None, None]:
+    with patch("personal_finances.user.jwt_utils.datetime") as mock:
```

Comment:
Are we modifying jwt_utils in the pull request? Couldn't find the reason to add this patch.

Reply:
changes not intended. I will removed it

Reply:
There's a comment from Leonardo:
[Mock datetime on jwt ultils test to avoid break](https://github.com/in-fin-neat/in-fin-neat-core/pull/52/commits/5926206f69759235454110f01eb3fe94a0435067)